packages/stores/src/stores/FeatureFlagsStore.ts

```diff
@@ -0,0 +1,30 @@
+import BaseStore from './BaseStore';
+
+const FLAGS = {
+    foo: false,
+    bar: false,
+    baz: false,
+};
```

Comment:
This is the only place we need to update for adding/removing a flag. All the types and variables are driven from this object.

If we add a new flag like `wallet` then the `useFeatureFlags` hook will expose a `is_wallet_enabled` variable for devs to use.

If we remove a flag from this list, Then TS will complain for every use of that flag in the code, This is useful for cleaning up a feature flag after a while.